examples/blog/schema.ts

```diff
@@ -0,0 +1,37 @@
+import { createSchema, list } from '@keystone-next/keystone/schema';
+import { select, relationship, text, timestamp } from '@keystone-next/fields';
+
+export const lists = createSchema({
+  Post: list({
+    fields: {
+      title: text({ isRequired: true }),
+      status: select({
+        dataType: 'enum',
+        options: [
+          { label: 'Draft', value: 'draft' },
+          { label: 'Published', value: 'published' },
+        ],
+        ui: {
+          displayMode: 'segmented-control',
+        },
```

Comment:
We're going to have a dedicated feature example for the `select` field type where we can demo use of `displayMode`.

```suggestion
```